third_party/chromium/media/formats/mp4/box_definitions.h

```diff
@@ -378,6 +378,44 @@ struct MEDIA_EXPORT OpusSpecificBox : Box {
   uint32_t sample_rate;
 };
 
+#if defined(STARBOARD)
+struct MEDIA_EXPORT IamfSpecificBox : Box {
+  DECLARE_BOX_METHODS(IamfSpecificBox);
+
+  enum IamfObuType {
```

Comment:
nit: type declaration before functions (i.e. DECLARE_BOX_METHODS above), and functions (e.g. `ReadSequenceHeader()`) before member variables.